### DISCUSSION TITLE
CLI command to render manifests

### DISCUSSION THREAD

**Discussion by pkalemba** [argoproj/argo-cd#16228] [Ideas]
Hi!

I'm thinking about a CLI command that will behave like dry-run, so instead of pushing changes to kube-API it should make YAML manifests rendered with all configurations in "app" to the console. 

It would be handy during validation processes.

**Comment by crenshaw-dev** [comment]
Does this command do what you need? https://argo-cd.readthedocs.io/en/stable/user-guide/commands/argocd_app_manifests/

**Reply by pkalemba** [reply]
yes and , no :<

I wanted to have the possibility to generate manifests before I make changes in argocd-app...
But I think I see problem....
Without pushing it to argocd-server I'm not able to generate anything, CLI has no plugins, and so on...
